src/System.Data.Common/ref/System.Data.Common.csproj

```diff
@@ -1,15 +1,20 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <PropertyGroup>
-    <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
-  </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Data.Common.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="project.json" />
+    <ProjectReference Include="..\..\System.Collections.NonGeneric\ref\System.Collections.NonGeneric.csproj" />
```

Comment:
>..\..\ [](start = 31, length = 6)

I wonder if we should use a property for the root of the corefx/src folder instead, since if for some reason we decide to change folder structure for refs, then all of the references will be broken.